src/sentry/tasks/post_process.py

```diff
@@ -206,10 +206,10 @@ def handle_owner_assignment(job):
                     if owners and not owners_exists:
                         try:
                             handle_group_owners(project, group, owners, owner_source)
-                        except Exception:
-                            logger.exception("Failed to store group owners")
-        except Exception:
-            logger.exception("Failed to handle owner assignments")
+                        except Exception as e:
+                            raise Exception("Failed to store group owners") from e
+        except Exception as e:
+            raise Exception("Failed to handle owner assignments") from e
```

Comment:
I wonder if we can get rid of this and let the try..catch on the pipeline catch the inner exceptions?

Reply:
we can just wanted to preserve original exception message (in case there's some log filter on it). if it's not important I can just pull them all

Reply:
Yeah. Not sure if we're especially filtering on these particular messages. But these messages don't seem particularly useful now that we do the try..catch and log when we execute the pipeline.

Reply:
sounds good. I'll just pull them